### DISCUSSION TITLE
How can I contribute

### DISCUSSION THREAD

**Discussion by GuilhermeDeLucca** [LiveHelperChat/livehelperchat#1897] [General]
I would also like to comment and I'm sorry for the boldness to say after all your documentation is very complete! And who am I to suggest an improvement, but I believe I could make the following change in
https://doc.livehelperchat.com/docs/development/remove-index-php/

change 
RewriteRule !\.(gif|jpe?g|png|bmp|css|js|xml|html|json|svg|ico|mp3|wasm|ogg|wav|ogv|swf|map|flv|otf|woff2|woff|eot|ttf)|var(.+)storage.pdf(.+)\.pdf$ index.php
To
RewriteRule !\.(gif|jpe?g|ico|png|bmp|css|js|xml|html|json|svg|ico|mp3|wasm|ogg|wav|ogv|swf|map|flv|otf|woff2|woff|eot|ttf)|var(.+)storage.pdf(.+)\.pdf$ index.php

and about using RewriteRule 

A note warning RewriteRule must be released in Apache and dependent on apache's vhost configuration:

<Directory /var/www/html/web_lhc>
                        AllowOverride All
</Directory>

Despite also understanding that the server part has nothing to do with the system and would not be pertinent to the documentation. But your documentation is so well done that this footnote was worth it.

**Comment by remdex** [comment]
Hi,

You can just create a pull request here. 
https://github.com/LiveHelperChat/doc

Also this one is related
https://github.com/orgs/LiveHelperChat/discussions/1888
